applications/Liberland_judiciary.md

```diff
@@ -0,0 +1,488 @@
+# Liberland Blockchain - the Judiciary
+
+- **Team Name:** Liberland Blockchain Team
+- **Payment Address:** fiat or, possibly, BTC
```

Comment:
BTC is not an option. Please check the application template for options and how to share your banking details with us in case of fiat.